pkg/deploy/controller/deploymentconfig/controller.go

```diff
@@ -247,6 +254,11 @@ func (c *DeploymentConfigController) reconcileDeployments(existingDeployments *k
 				glog.V(4).Infof("Updated deployment %q replica annotation to match current replica count %d", deployutil.LabelForDeployment(&deployment), newReplicaCount)
 			}
 		}
+		deployments = append(deployments, &deployment)
 	}
-	return nil
+	// TODO: Use a separate call for updating the status of a DC
+	// https://github.com/openshift/origin/issues/6465
+	config.Status.Replicas = kdeputil.GetReplicaCountForRCs(deployments)
+	_, err := c.osClient.DeploymentConfigs(config.Namespace).Update(config)
```

Comment:
I think you're right- shouldn't we be reassigning config local after the update on L220?